### PR TITLE
Updated data tiering section

### PR DESCRIPTION
**Pull request by mfreed**
- Rewrote section under /using-timescaledb/data-tiering
- Added back into (and slightly reorganized) page index
- Reference docs in /api#move_chunk

### CODE REVIEW

File: using-timescaledb/data-tiering.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Data Tiering 
+
+TimescaleDB includes the ability to perform data tiering using a built-in
```
Comment (svenklemm) [inline using-timescaledb/data-tiering.md:3]: do we refer to other timescaledb functions as built-in? sounds a bit weird

File: using-timescaledb/data-tiering.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+function [`move_chunk`][api-move-chunk].  This action
```
Comment (svenklemm) [inline using-timescaledb/data-tiering.md:4]:
```suggestion
function [`move_chunk`][api-move-chunk].  This function
```

File: using-timescaledb/data-tiering.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+read/write workload against recent data (in the default tablespace).
+
+## Creating a Tablespace
+
+The [`move_chunk`][api-move-chunk] function requires multiple tablespaces set up in PostgreSQL, so let's
+start with a quick review of how this works.
+
+First, add a storage mount that will serve as a home for your new tablespace. This
+process will differ based on how you are deployed, but your system administrator
+should be able to arrange setting up the mount point. The key here is to backend
```
Comment (svenklemm) [inline using-timescaledb/data-tiering.md:41]:
```suggestion
should be able to arrange setting up the mount point. The key here is to back
```

File: using-timescaledb/data-tiering.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+To create a [tablespace][] in Postgres:
+
+```sql
+CREATE TABLESPACE history
+OWNER postgres
+LOCATION '/mnt/history':
+```
+
+Here we are creating a tablespace called `history` that will be in this case
```
Comment (svenklemm) [inline using-timescaledb/data-tiering.md:52]:
```suggestion
Here we are creating a tablespace called `history` that will be
```

File: using-timescaledb/data-tiering.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+chunks and their indexes.
+
+## Move Chunks :community_edition: [](move_chunks)
+
+Now that we have set up a new, empty tablespace, we can move individual chunks
+from their default tablespace to this new secondary tablespace.  The move
+chunks command also allows you to move indexs belonging to those chunks to this
+secondary tablespace (or another on).
+
+In addition, the [`move_chunk`][api-move-chunk] functionality alow provides the
```
Comment (svenklemm) [inline using-timescaledb/data-tiering.md:65]:
```suggestion
In addition, the [`move_chunk`][api-move-chunk] functionality provides the
```

**Review by svenklemm — APPROVED** [review]
Couple typos and minor suggestions

File: using-timescaledb/data-tiering.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+To determine which chunks to move, we can list chunks that fit a specific
+criteria.  For example, to identify chunks older than two days:
+
+```sql
+SELECT show_chunks('conditions', older_than => INTERVAL '2 days');
+```
+
+In this example, we will move `_timescaledb_internal._hyper_1_4_chunk` along
+with its index over to `history` using the following command, while reordering
+the chunk based on its time index:
```
Comment (solugebefola) [inline using-timescaledb/data-tiering.md:79]:
```suggestion
Or to, for example, move `_timescaledb_internal._hyper_1_4_chunk` along
with its index over to `history`, while reordering the chunk based on its time index:
```

File: using-timescaledb/data-tiering.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+Once this successfully executes, we can verify that our chunk now lives on the
+`history` tablespace by querying `pg_tables` to list all of the chunks that
+are on `history`:
+
+```sql
+SELECT tablename from pg_tables
+  WHERE tablespace = 'history' and tablename like '_hyper_%_%_chunk';
+```
+
+As you will see, the target chunk is now listed as residing on `history`; we
```
Comment (solugebefola) [inline using-timescaledb/data-tiering.md:99]: Here and directly below, I feel like there should be example output.

File: using-timescaledb/data-tiering.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+SELECT move_chunk(
+  chunk => '_timescaledb_internal._hyper_1_4_chunk',
+  destination_tablespace => 'history',
+  index_destination_tablespace => 'pg_default',
+  reorder_index => '_timescaledb_internal._hyper_1_4_chunk_netdata_time_idx'
+);
+```
+
+You could perform the opposite as well (keeping the data in `pg_default` but
+moving the index to `history`), or you can setup a third tablespace
```
Comment (solugebefola) [inline using-timescaledb/data-tiering.md:133]:
```suggestion
moving the index to `history`), or setup a third tablespace
```

**Review by solugebefola — COMMENTED** [review]
A few suggestions and some corrections of typos, etc.

File: using-timescaledb/data-tiering.md (hunk 7 above)
Comment (mfreed) [reply using-timescaledb/data-tiering.md:95]: Don't disagree, but going to skip for now for time...

**Pull request merged**